server/src/main/proto/spine/server/integration/integration_event.proto

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright 2016, TeamDev Ltd. All rights reserved.
+//
+// Redistribution and use in source and/or binary forms, with or without
+// modification, must retain the above copyright notice and the following
+// disclaimer.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+// "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+// LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+// A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+// OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+// SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+// LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+// DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+// THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+syntax = "proto3";
+
+package spine.server.integration;
+
+option java_package = "org.spine3.server.integration";
+option java_multiple_files = true;
+option java_outer_classname = "IntegrationEventProto";
+option java_generate_equals_and_hash = true;
+
+import "google/protobuf/any.proto";
+
+import "spine/base/event.proto";
+import "spine/base/response.proto";
+
+message IntegrationEvent {
+    // The message of the event message wrapped into `Any`.
+    google.protobuf.Any message = 1;
+
+    // The context of the event.
+    spine.base.EventContext context = 2;
+
+    // The name of the bounded context, which generated the event.
+    string source = 3;
+}
+
+service IntegrationEventSubscriber {
+
+    rpc Notify(IntegrationEvent) returns (spine.base.Response);
```

Comment:
As I understand, BoundedContext should implement this service interface. Why `notify` and not `post`?